lesson8-UnitTesting/FizzBuzz/tests/FizzBuzzTests/Tests.cs

```diff
@@ -0,0 +1,33 @@
+using FizzBuzzProj;
+using NUnit.Framework;
+using System;
+
+namespace FizzBuzzTests
+{
+    public class Tests
```

Comment:
Test class name should reflect what component it tests, so I'd name it FizzBuzzTests. Also looks like `[TestFixture]` attribute is missing on the class. Tests still may work but this class may not be taken into account in case you e.g. decide to create CI for the project.

Reply:
done